packages/cli/src/lib/decorators/preconditions/ng.spec.ts

```diff
@@ -1,7 +1,7 @@
 jest.mock('../../utils/process');
```

Comment:
Simplified those tests to use snapshot. To have os-agnostic snapshot I mock everything and tada.